pkgs/by-name/pa/package-version-server/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "package-version-server";
+  version = "0.0.7";
+
+  src = fetchFromGitHub {
+    owner = "zed-industries";
+    repo = "package-version-server";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/YyJ8+tKrNKVrN+F/oHgtExBBRatIIOvWr9mAyTHA3E=";
+  };
+
+  cargoHash = "sha256-aO6d7NcWZtLMH2/2jcsD8vpDO+C2exMwxrLVFH3bsP0=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  doCheck = false;
```

```suggestion
  doCheck = lib.versionOlder version "0.0.8";
```
Opened https://github.com/zed-industries/package-version-server/pull/2.

Though I am not sure why they don't run their tests...

---

I didn't know about lib.versionOlder, this is so much better, thank you.